scripts/epa/airdata/import_config.txt

```diff
@@ -0,0 +1,14 @@
+import: {
+  import_name: "EPACriteriaAirPollutants"
+  provenance_url: "https://aqs.epa.gov/aqsweb/airdata/download_files.html "
+  mcf_url: "/cns/jv-d/home/datcom/v3_resolved_mcf/epa/airdata/nodes.mcf@100"
+  start_date_in_kg: "1980-01-01"
+  end_date_in_kg: "2021-02-28"
+  end_date_in_source: "2021-02-28"
+  release_frequency: "P6M"
+  next_release_date: "2021-11"  # ESTIMATED
+  table {
+    mapping_path: "/cns/jv-d/home/datcom/v3_mcf/epa/airdata/EPA_AirQuality.tmcf"
+    csv_path: "/cns/jv-d/home/datcom/v3_mcf/epa/airdata/EPA_AirQuality.csv"
+  }
```

Comment:
Sorry, I failed to catch this before, but this manifest is actually different from the one in google3.  It has different fields, used for refreshes and would not include cns references. For example, https://github.com/datacommonsorg/data/blob/master/scripts/covid19_india/cases_count_states_data/manifest.json

I suggest just skipping it...

Reply:
removed